scripts/configure_release_please/release-please-config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Is this file only here for testing? If so, could we move it under test_resources/?

Reply:
Fixed in 5c77eba